docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -48,14 +52,14 @@ haproxy-bundle successfully restarted
     * haproxy-bundle-podman-2   (ocf:heartbeat:podman):  Started controller-2
 ----
 
-. Confirm that no process is bound to 8080:
+. Confirm that no process is connected to the 8080 port:
 +
 ----
 [root@controller-0 ~]# ss -antop | grep 8080
 [root@controller-0 ~]#
 ----
 +
-The {object_storage_first_ref} CLI fails at this point:
+The {object_storage_first_ref} CLI fails to establish the connection. This is expected:
```

Comment:
"You can expect the {object_storage_first_ref} CLI to fail to establish the connection:"